database/pgsql/migrations/00019_add_vuln_state_rhel.go

```diff
@@ -6,7 +6,7 @@ func init() {
 	RegisterMigration(migrate.Migration{
 		ID: 19,
 		Up: migrate.Queries([]string{
-			`ALTER TABLE vuln_v2 ADD COLUMN IF NOT EXISTS package_resolution_state TEXT NOT NULL DEFAULT ''`,
+			`ALTER TABLE vuln_v2 ADD COLUMN IF NOT EXISTS resolution_state TEXT`,
```

Comment:
no need to make this `NOT NULL`. Also, change the name to something more concise

Reply:
One of the tests failed because there's no "NOT NULL". I am guessing it's because not all package has an resolution state.

Reply:
Yeah, there are `1777588` rows in `vuln_v2` with empty `resolution_state` :D . I don't think we need to worry about `NOT NULL` since Go would turn it into an empty string upon insert and read